app/components/pages/Contact.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Contact extends React.Component {
```

Comment:
## Code Quality - Nitpicks

Export classes as you create them. One liners - Neat, clean and less code